testing/infra/terraform/modules/ec_deployment/deployment.tf

```diff
@@ -15,6 +15,7 @@ resource "ec_deployment" "deployment" {
   version                = data.ec_stack.deployment_version.version
```

Comment:
What do you think if we raise an internal issue so we can track the consumers and then if we add some tool around this could benefit from?

Reply:
There are some tools we have made to create ephemeral or permanent cloud deployments (in ESS, ECE, ECK), the `oblt-cli` tool is one of them, and helps users to create their deployments without the hassle to know much. Since it's actively used and support different providers, configurations, we could ideally make the Benchmark for the `apm-server` a consumer. Though, it's an internal tool, so only Elasticians can use it.

Reply:
The idea was to do nothing but track this project as a potential candidate to use the tool in the future